state/cloudcredentials.go

```diff
@@ -183,7 +183,7 @@ func cloudCredentialDocID(tag names.CloudCredentialTag) string {
 	return fmt.Sprintf("%s#%s#%s", tag.Cloud().Id(), tag.Owner().Id(), tag.Name())
 }
 
-func (c cloudCredentialDoc) cloudCredentialTag() (names.CloudCredentialTag, error) {
+func (c cloudCredentialDoc) CloudCredentialTag() (names.CloudCredentialTag, error) {
```

Comment:
methods on the state docs should not be exported - the content is accessed via the wrapped struct into which the doc is embedded. you may need a add a method to state.Credential

Reply:
Ok